.golangci.yaml

```diff
@@ -25,4 +26,7 @@ linters:
     - unused
     - varcheck
     - whitespace
-
+linters-settings:
+  funlen:
+    lines: 96
```

Comment:
Where do these values come from?
Would it be better to adjust these to match the current code rather than mark some of the existing functions not to be checked?

Reply:
OK, will align those values with current code (lines: 188, statements: 60)